src/chains/ethereum/console.log/tests/index.test.ts

```diff
@@ -30,6 +30,7 @@ describe("@ganache/console.log", () => {
   let from: string;
 
   before("set up a ganache provider", function () {
+    this.timeout(10000);
```

Comment:
Because Ethereum ganache is lazy loaded, the first time we run it it might take a while, so I added this timeout. Previously it would take the same about of time, but it was just doing it before the tests started running.